src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCONT = 50;
```

Comment:
There is a typo in the constant name 'LONG_TERM_DISCONT'. It should be 'LONG_TERM_DISCOUNT' to correctly represent the discount for long term rentals.